pkg/client/cache/watch_cache_test.go

```diff
@@ -108,6 +109,28 @@ func TestEvents(t *testing.T) {
 	store := NewWatchCache(5)
 
 	store.Add(makeTestPod("pod", 2))
+
+	// Test for Added event.
+	{
+		result, err := store.GetAllEventsSince(2)
```

Comment:
nit: it isn't clear if there is some special significance to the index, if not it would be clearer if you just looped and verified the edge indices as well.